hit-on.cabal

```diff
@@ -81,4 +81,20 @@ executable hit
   ghc-options:         -Wall
                        -threaded
                        -rtsopts
-                       -with-rtsopts=-N
\ No newline at end of file
+                       -with-rtsopts=-N
+
+
+test-suite hit-on-test
+  type:                exitcode-stdio-1.0
+  hs-source-dirs:      test
+  main-is:             Main.hs
+  build-depends:       base
+                     , text
+                     , hspec
+                     , github
+                     , hit-on
+  other-modules:       
```

Comment:
This section is empty, so you can remove it. Later, when we will have many tests, we will, of course, add new modules.